### ISSUE TITLE
Scaffolding inertia-react doesn't work

### ISSUE THREAD

**Issue by cherifGsoul** [ishanvyas22/asset-mix#25]
**Describe the bug**
I'm using `cakephp-inertiajs`:

```shell
bin/cake asset_mix generate inertia-react
```

Output:
```shell
Error: "inertia-react" is not a valid value for preset. Please use one of "bootstrap, vue, react, inertia-vue"
```

**To Reproduce**
1. Install and load the plugin (I did through `cakephp-inertiajs`)
2. Run `bin/cake asset_mix generate inertia-react`

**Expected behavior**
Scaffolded `inertia-react` app in assets directory. 

**Screenshots**
![Screen Shot 2021-03-18 at 1 29 57 PM](https://user-images.githubusercontent.com/109013/111626251-1210f700-87ee-11eb-91f0-f44c6e5f1843.png)

Not sure if this is related to `cakephp-inertiajs` (asset-mix not upgraded maybe), I didn't make a check yet.

**Comment by cherifGsoul** [comment]
Seems that the package is updated in `cake-inertiajs`
![Screen Shot 2021-03-18 at 1 37 33 PM](https://user-images.githubusercontent.com/109013/111627235-26092880-87ef-11eb-8903-a79282e248af.png)

**Comment by cherifGsoul** [comment]
Maybe @markstory [PR](https://github.com/ishanvyas22/asset-mix/commit/d843639b5e6b9d32ec8025653f6502c340578800) needs to be released!

**Comment by ishanvyas22** [comment]
@cherifGsoul I've released @markstory's PR in [1.3.0](https://github.com/ishanvyas22/asset-mix/releases/tag/1.3.0). Can you please update the [`cakephp-inertiajs`](https://packagist.org/packages/ishanvyas22/cakephp-inertiajs) plugin with `--with-dependencies` option and check? It should be fixed now.

**Comment by cherifGsoul** [comment]
@ishanvyas22 Thank you 🙏🏻
